README.Md

```diff
@@ -24,7 +24,9 @@ might be supported in a near future.
 
 Getting Started
 ===============
-There is much more information, including a more complete "Getting Started Guide," extensive documentation, examples, and contact information available on the [AtomVM](https://atomvm.net) project website.
+There is much more information, including a more complete (["Getting Started Guide"](https://www.atomvm.net/doc/master/getting-started-guide.html)), ([extensive documentation](https://www.atomvm.net/doc/master/build-instructions.html)), ([examples](https://www.atomvm.net/sample-code)), detailed ([build instructions](https://www.atomvm.net/doc/master/build-instructions.html)) and ([contact information](https://www.atomvm.net/contact)) available on the [AtomVM](https://atomvm.net) project website.
```

Comment:
What would you think of pointing the example to the atomvm_examples repo instead?